python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -6787,8 +6794,9 @@ tables:
     datatype: double
     ivoa:unit: d
 - name: SSSource
-  description: LSST-computed per-source quantities. 1:1 relationship with DIASource.
+  description: LSST-computed per-source quantities. 1:1 relationship with DiaSource.
     Recomputed daily, upon MPCORB ingestion.
```

Comment:
Probably makes sense to remove the "Recomputed daily" sentence in the DP1 context.

Reply:
Removed; I'd already taken out the corresponding text in the MPCORB description.  Good catch!